framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -405,18 +405,17 @@ player_result_t MediaPlayerImpl::setVolume(int vol)
 	return ret;
 }
 
-void MediaPlayerImpl::setVolumePlayer(int vol, player_result_t &ret)
+void MediaPlayerImpl::setVolumePlayer(uint8_t vol, player_result_t &ret)
 {
-	int vol_max = -1;
+	uint8_t vol_max;
 	medvdbg("MediaPlayer Worker : setVolume\n");
 
-	vol_max = get_max_audio_volume();
-	if (vol < 0 || vol > vol_max) {
+	if (get_max_audio_volume(&vol_max) != AUDIO_MANAGER_SUCCESS) {
```

Comment:
i don't understand this logic.
if you wanna get max_volume, you need to compare beteern max_vol and settting volume value.
you're missing this point.

Reply:
Oops~ You are right. I'll revert & modify this..